k9mail/src/main/java/com/fsck/k9/activity/setup/AccountSettings.java

```diff
@@ -697,6 +707,11 @@ public boolean onPreferenceClick(Preference preference) {
                 return true;
             }
         });
+
+        if (startInOpenPgp) {
```

Comment:
You probably need to add a check for `savedInstanceState == null`. Otherwise orientation changes will kick us into the OpenPGP screen regardless which `PreferenceScreen` was active before.